src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6012,9 +6008,10 @@ namespace InternalHeatGains {
         } // ZoneITEq calc loop
 
         // Zone-level sensible heat index
-        for (Loop = 1; Loop <= NumOfZones; ++Loop) {
-            if (ZoneSumToutMinusTSup(Loop) != 0.0) {
-                ZnRpt(Loop).ITEqSHI = ZoneSumTinMinusTSup(Loop) / ZoneSumToutMinusTSup(Loop);
+        for (Loop = 1; Loop <= NumZoneITEqStatements; ++Loop) {
+            int ZN = ZoneITEq(Loop).ZonePtr;
+            if (ZoneSumToutMinusTSup(ZN) != 0.0) {
+                ZnRpt(ZN).ITEqSHI = ZoneSumTinMinusTSup(ZN) / ZoneSumToutMinusTSup(ZN);
```

Comment:
much better!

Reply:
Do these arrays really need to be allocated to NumOfZones ? Could just save that information with each ITE object?

    Array1D<Real64> ZoneSumTinMinusTSup(NumOfZones)
    Array1D<Real64> ZoneSumToutMinusTSup(NumOfZones)

Reply:
Yeah, that would be better. I'll address that.

Reply:
This was addressed here: d51577b49f74f35ed74d393323e8d73027baf6fe